### PR TITLE
Codefresh wait to have build queue

### PR DESCRIPTION
**Pull request by goruha**
## What
* Provide Codefresh wait target

## Why
* Allow guaranteed builds order

### CODE REVIEW

File: bin/codefresh-wait.sh [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+function is_not_next() {
+	local id=$1
+	local branch=$2
+	local piplines_names=($PIPELINES)
+
+	next_id=$(${CODEFRESH_CLI} get builds ${piplines_names[@]/#/--pipeline-name } \
+		--branch ${branch} \
+		--status ${STATUS}\
+		--limit ${LIMIT} \
+		-o id | tac | head -1)
```
Comment (osterman) [inline bin/codefresh-wait.sh:18]: `tac | head -1` ~ `tail -1`, no?

File: bin/codefresh-wait.sh [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+STATUS=${STATUS:-running}
+PIPELINES=${PIPELINES:-}
+
+
+function is_not_next() {
+	local id=$1
+	local branch=$2
+	local piplines_names=($PIPELINES)
+
+	next_id=$(${CODEFRESH_CLI} get builds ${piplines_names[@]/#/--pipeline-name } \
```
Comment (osterman) [inline bin/codefresh-wait.sh]:
```suggestion
	next_id=$(${CODEFRESH_CLI} get builds ${piplines_names[@]/#/--pipeline-name} \
```

File: bin/codefresh-wait.sh (hunk 1 above)
Comment (goruha) [reply bin/codefresh-wait.sh:18]: Done

**Comment by goruha** [comment]
The feature can be solved by commands in codefresh.yaml

**Comment by goruha** [comment]
```
  wait:
    title: Wait
    stage: Prepare
    image: codefresh/cli
    commands:
      - >-
          export QUEUE_HEAD_ID=$( \
            codefresh get builds \
              --pipeline-name=integration-create \
              --pipeline-name=integration-destroy \
              --branch ${{CF_BRANCH}} \
              --status running \
              --limit 1000 -o id | \
            tail -1 | tr -dc '[[:print:]]' \
          )
      - echo "|${QUEUE_HEAD_ID}| == |${CF_BUILD_ID}|"
      - 'if [ "${QUEUE_HEAD_ID}" != "${CF_BUILD_ID}" ]; then exit 1; fi'
    retry:
      maxAttempts: 10
      delay: 20
      exponentialFactor: 1.1
```

**Pull request closed without merging**